packages/plugin-ext/doc/how-to-add-new-custom-plugin-api.md

```diff
@@ -62,94 +70,70 @@ declare module '@bar/foo' {
 ## Implement your plugin API provider
 
 In our example, we aim to provide a new API object for the backend.
-Theia expects that the `backendInitPath` that we specified in our API provider is a function called `provideApi` that follows the `ExtPluginApiBackendInitializationFn` signature.
+Theia expects that the `backendInitPath` that we specified in our API provider exports an [InversifyJS](https://inversify.io) `ContainerModule` under the name `containerModule`.
+This container-module configures the Inversify `Container` in the plugin host for creation of our API object.
+It also implements for us the customization
+For export of the API to headless plugins, use the `headlessInitPath` with a `provideApi` function of `ExtPluginApiHeadlessInitializationFn` type.
```

Comment:
Not sure if we need the type info here since we do not show the same level of detail for the other plugin types, especially not the backendInitPath which is the one we actually showcase.

Reply:
This is a relic from before my second pass that rewrote the example in terms of the new `ContainerModule` mechanism that supersedes the `provideApi` function!